src/main/webapp/app/entities/study/study-dialog.component.ts

```diff
@@ -124,6 +124,14 @@ export class StudyDialogComponent implements OnInit {
         }
         return option;
     }
+
+    getTitle() {
+        if ( this.study.id != null ) {
```

Comment:
Thinking that there could be a helper method here -- something like `this.isNew()`, since this logic is used in a few places. Will create a new issue for this.

Reply:
#58